tests/sentry/issues/test_occurrence_consumer.py

```diff
@@ -37,7 +37,7 @@ def get_test_message(
 
     if include_event:
         payload["event"] = {
-            "event_id": uuid.uuid4().hex,
+            "event_id": str(uuid.uuid4().hex),
```

Comment:
not sure if there is a better or more correct way to do this

Reply:
Not sure if it matters, but I use `str(uuuid.uuid4())`. It doesn't look like you need to validate the format of the string anyways though so basically any string should work here?

Reply:
right yeah. I just meant casting it to `str`

Reply:
`.hex` should already be a string https://docs.python.org/3/library/uuid.html#uuid.UUID.hex

Reply:
it was fighting me. will try again